### PR TITLE
Data group

### PR DESCRIPTION
**Pull request by dianaboiangiu**

### CODE REVIEW

File: insitu/templates/data_group/list.html [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+      </table>
+    </div>
+
+   <div class="col-lg-2">
+      <div class="panel panel-default">
+        <div class="panel-heading">Filters</div>
+        <div class="panel-body">
+
+          <div class="form-group">
+            <label for="name">Name:</label>
```
Comment (andreeadima) [inline insitu/templates/data_group/list.html]: Delete this filter

File: insitu/templates/data_group/add.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+{% block scripts %}
+  <script src="{% static 'js/jquery-3.2.1.min.js' %}"></script>
+  <script src="{% static 'js/select2.min.js' %}"></script>
+  <script>
+    $(document).ready(function () {
+      $('select').select2();
+    });
+  </script>
+{% endblock %}
```
Comment (andreeadima) [inline insitu/templates/data_group/add.html]: Missing enter

File: insitu/templates/data_group/detail.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            <p>{{ essential_climate_variable }}</p>
+          {% endfor %}
+        </td>
+      </tr>
+      {% endif %}
+
+      </tbody>
+    </table>
+  </div>
+{% endblock %}
```
Comment (andreeadima) [inline insitu/templates/data_group/detail.html]: Missing enter

File: insitu/templates/data_group/detail.html [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+      <tr class="row">
+        <th class="col-md-4 col-sm-6">Inspire themes</th>
+        <td class="col-md-8 col-sm-6">
+          {% for inspire_theme in data_group.inspire_themes.all %}
+            <p>{{ inspire_theme }}</p>
+          {% endfor %}
+        </td>
+      </tr>
+      {% if data_group.essential_climate_variables.all|length %}
+      <tr class="row">
```
Comment (andreeadima) [inline insitu/templates/data_group/detail.html]: Indent

File: insitu/templates/data_group/edit.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+{% block style %}
+  <link rel="stylesheet" type="text/css"
+        href="{% static 'css/select2.min.css' %}"/>
+  <link rel="stylesheet" type="text/css"
+        href="{% static 'css/select2-bootstrap.min.css' %}"/>
+{% endblock %}
+
+{% block title %}
+  Edit data group {{ object.name }}
```
Comment (andreeadima) [inline insitu/templates/data_group/edit.html]: Name on another line

File: insitu/tests/test_data_group.py [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+
+    def test_edit_data_group(self):
+        data_group = base.DataGroupFactory()
+        data = self._DATA
+        resp = self.client.post(
+            reverse('data_group:edit', kwargs={'pk': data_group.pk}),
+            data)
+        self.assertEqual(resp.status_code, 302)
+        self.check_single_object(models.DataGroup, data)
```
Comment (andreeadima) [inline insitu/tests/test_data_group.py]: Missing new line

**Comment by dianaboiangiu** [comment]
Requested changes made.

**Pull request merged**